c1/src/lib.rs

```diff
@@ -1,11 +1,13 @@
 #![allow(dead_code)]
 use near_sdk::{
   borsh::{self, BorshDeserialize, BorshSerialize},
-  env, ext_contract, near_bindgen, wee_alloc, Promise, PromiseOrValue,
+  env, ext_contract, near_bindgen,
+  Promise, PromiseOrValue,
 };
 
+#[cfg(target = "wasm32")]
```

Comment:
oh nifty lil hack. What exactly does this do?

Reply:
I know I answer in your call but I thought I'd put this here too.  This will only compile the following block of code if the target of the compilation is Wasm and the simulation tests target your host architecture.